mandatory/3-playing-computer.js

```diff
@@ -7,36 +7,49 @@
   Answer the following questions:
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
+ because in line 35 b is not defined
+
   2. Remove the line that throws the error.
+done it
+
   3. What is printed to the console?
+
+  
   4. How many times is "f1" called?
+  2 times
+
   5. How many times is "f2" called?
+  3 times
+
   6. What value does the "a" parameter take in the first "f1" call?
+the value is 8
```

Comment:
Look at this again - the first parameter in the f1 method is called a, but what value is being passed in by the code in the for loop?
f1(i, a); (I think this code is deliberately meant to be confusing)